c7n/resources/ses.py

```diff
@@ -34,3 +35,15 @@ class resource_type(TypeInfo):
         'describe': DescribeConfigurationSet
     }
 
+
+@resources.register('ses-email-identity')
+class SESEmailIdentity(QueryResourceManager):
+    class resource_type(TypeInfo):
+        service = 'sesv2'
+        enum_spec = ('list_email_identities', 'EmailIdentities', None)
+        detail_spec = ('get_email_identity', 'EmailIdentity', 'IdentityName', None)
+        name = id = 'IdentityName'
+        arn_type = 'identity'
+        universal_taggable = object()
```

Comment:
Looks like resourcegrouptagging is not supported on this resource. Traceback
```
  File "/Users/pratyushmishra/development/cloud-custodian/c7n/tags.py", line 1223, in universal_retry
    raise Exception("Resource Tag Errors %s" % (errors))
Exception: Resource Tag Errors {'arn:aws:sesv2:us-east-1:644160558196:identity/c7ntest@gmail.com': 'InvalidParameterException'}
```
You may want to add tag/remove-tag/mark-for-op and marked-for-op actions separately outside of resourcegrouptagging

Reply:
[update] - so looks like resourcegrouptagging api is supported but the arn we're constructing here is not right. The arn should be - `arn:aws:ses:us-east-1:644160558196:identity/c7ntest@gmail.com`